front/app/containers/Admin/projects/project/nativeSurvey/AccessRightsNotice/index.tsx

```diff
@@ -32,101 +26,36 @@ const AccessRightsNotice = ({
   phaseId,
   handleClose,
 }: AccessRightsNoticeProps) => {
-  const localize = useLocalize();
   const { formatMessage } = useIntl();
```

Comment:
Simplified this notice in the form builder a bit. Explaining all the nuances was too complicated so I just link to the access rights basically